src/components/elements/Inputs/Input/Input.tsx

```diff
@@ -79,6 +79,7 @@ const Input = forwardRef(
       ...inputProps
     } = inputWrapperProps;
     const id = useId();
+    const customVariantClasses = customVariant ? customVariant : {};
```

Comment:
:art: This would be better either as `const customVariantClasses = customVariant?? {}` or line 117 could simply be `customVariant ?? {}` since this const isn't used anywhere else.

Reply:
Done.